### PR TITLE
Lead capture improvements

### PR DESCRIPTION
**Pull request by dtuite**
https://app.shortcut.com/larder/story/20833

Preview: https://deploy-preview-1296--roadie.netlify.app/request-demo/

### CODE REVIEW

**Review by roadgork — APPROVED** [review]

File: src/pages/request-demo.js [hunk 1]
```diff
@@ -68,9 +68,15 @@ const RequestDemo = ({ data, location }) => {
           <Headline size="medium" className="leading-snug text-orange-600 relative z-10">
             Discover Roadie
           </Headline>
+
           <h2 className="text-lg mt-3 lg:text-xl xl:text-xl font-bold relative z-10">
             Get a private Roadie demo from a Backstage expert.
           </h2>
+
+          <p>
+            Ask us about security, costs, how you can customize Roadie and how integrations work.
```
Comment (punkle) [inline src/pages/request-demo.js:77]: I cant put a fine point on it, but this sentence seems a bit off.

Comment (punkle) [reply src/pages/request-demo.js:77]: How about "Ask us about security; costs; how you can customize Roadie; and how integrations work."

Comment (punkle) [reply src/pages/request-demo.js:77]: or: Ask us about security; costs; customization; and integrations.

Comment (dtuite) [reply src/pages/request-demo.js]: You're right. I've fixed it

**Pull request merged**